clang/test/SemaSYCL/intel-fpga-loops.cpp

```diff
@@ -304,6 +323,11 @@ void zoo() {
   // expected-error@+1 {{duplicate Intel FPGA loop attribute 'nofusion'}}
   [[intel::nofusion]] for (int i = 0; i != 10; ++i)
       a[i] = 0;
+
+  [[intel::loop_control_avg(2)]]
+  // expected-error@+1{{duplicate Intel FPGA loop attribute 'loop_control_avg'}}
+  [[intel::loop_control_avg(2)]] for (int i = 0; i != 10; ++i)
+      a[i] = 0;
```

Comment:
Could you add a test with disbale_loop_pipeline since no diagnostic is expected i believe.